docs/modelfile.md

```diff
@@ -89,6 +89,35 @@ FROM ./ollama-model.bin
 
 This bin file location should be specified as an absolute path or relative to the Modelfile location.
 
+### Build from unquantized bin file
+
+```
+FROM ./ollama-model-f16.bin AS Q4_0
```

Comment:
I'm worried about using the `AS` keyword here because it's used differently than the way you would use it with a Dockerfile, and it might be confusing. With Docker, AS would create a new context (and associated layers) which allow you to copy data from previously computed layers, or to create a `manifest list` with multiple architectures being generated from the same Modelfile.

Reply:
I'm open to ideas

Reply:
maybe:

```
FROM ./ollama-model-f16.bin QUANTIZE Q4_0
```

That way we could do multiple FROM lines in the Modelfile each with different quantization levels.

Reply:
The only thing I'm slightly on the fence with QUANTIZE is that it's jargon, but given you need to understand what Q4_0 or Q5_K_S means anyway, I think I'm OK with that. The other benefit is we can later add AS to specify the different layers in the manifest list.

Reply:
> That way we could do multiple FROM lines in the Modelfile each with different quantization levels.

This doesn't make sense to me since they would need the same base model

```
FROM ./ollama-model-f16.bin QUANTIZE Q4_0
FROM ./ollama-model-f16.bin QUANTIZE Q4_1
FROM ./ollama-model-f16.bin QUANTIZE Q4_K_S
FROM ./ollama-model-f16.bin QUANTIZE Q4_K_M
```

> The other benefit is we can later add AS to specify the different layers in the manifest list.

I don't see a parallel to Dockerfile's `FROM ... AS ...` where there will be multiple and different `FROM` instructions and you need to reference other base layers

A plus with `QUANTIZE` is there could be a default since the intent is clear. Something like `FROM ... QUANTIZE` can default to `Q4_0`, as an example

Reply:
Being from the same base model is fine. I don't think that's particularly confusing for users.

We will need to reference other parts of the create step if we introduce manifest lists. You may need to change the prompts. I think this could also impact the embeddings and possibly also with lora training (as opposed to just adding the lora layers). I'm still not 100% sure how this should work yet though.